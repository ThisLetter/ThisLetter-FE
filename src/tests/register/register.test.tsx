import { render } from "@testing-library/react";
import RegisterInputCheck from "../../components/molecules/register/RegisterInputCheck";

describe("<RegisterInputOrg />", () => {
  it("matches snapshot", () => {
    const utils = render(
      <RegisterInputCheck placeholder={"test"} id={"test"} onClick={() => {}} />
    );
    expect(utils.container).toMatchSnapshot();
  });
  it("shows the props correctly", () => {
    const utils = render(
      <RegisterInputCheck placeholder={"test"} id={"test"} onClick={() => {}} />
    );
    utils.getByPlaceholderText("test");
  });
});
