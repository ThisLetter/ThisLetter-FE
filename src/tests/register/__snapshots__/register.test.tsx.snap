// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterInputOrg /> matches snapshot 1`] = `
<div>
  <div
    class="flex mb-4"
  >
    <input
      class="border rounded flex-1 p-2 mr-4"
      id="test"
      placeholder="test"
      type="text"
    />
    <button
      class="border-solid border-2 border-black rounded-md bg-white px-4 py-2 flex justify-center align-center undefined"
      type="button"
    >
      확인하기
    </button>
  </div>
</div>
`;
